source/server/admin/stats_params.h

```diff
@@ -69,6 +69,7 @@ struct StatsParams {
   std::string filter_string_;
   std::shared_ptr<re2::RE2> re2_filter_;
   Utility::HistogramBucketsMode histogram_buckets_mode_{Utility::HistogramBucketsMode::NoBuckets};
+  Utility::HistogramMode histogram_mode_{Utility::HistogramMode::Histogram};
```

Comment:
it's definitely unexpected to see these 2 fields in this structure, and the second one only applies to prom.